.github/workflows/tests.yml

```
@@ -0,0 +1,48 @@
+          echo 'nameserver 8.8.4.4' | sudo tee /etc/resolv.conf
+          echo 'nameserver 9.9.9.10' | sudo tee -a /etc/resolv.conf
+          echo 'nameserver 77.88.8.8' | sudo tee -a /etc/resolv.conf
+          echo 'nameserver 168.95.1.1' | sudo tee -a /etc/resolv.conf
+      
+      - name: Run nslookup test
+        run: ./tests/nslookup
+
+  duplicate-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)